src/corehost/cli/runtime_config.cpp

```diff
@@ -124,8 +137,74 @@ bool runtime_config_t::parse_opts(const json_value& opts)
     m_portable = true;
 
     const auto& fx_obj = framework->second.as_object();
+
     m_fx_name = fx_obj.at(_X("name")).as_string();
-    m_fx_ver = fx_obj.at(_X("version")).as_string();
+
+    // Step #3: Use the values in the app's "framework" section
+    bool rc = parse_framework(fx_obj);
```

Comment:
Review note: we could just read "version" here instead instead of calling into `parse_framework`. However by calling into `parse_framework` we allow the other values (applyPatches, etc) to be specified and make the schema consistent with "additionalFrameworks"